sdk/python/setup.py

```diff
@@ -45,6 +45,7 @@
     install_requires=REQUIRES,
     packages=[
         'kfp',
+        'kfp.cli',
```

Comment:
I have already added it in a PR. You probably need a merge.

Reply:
GitHub does not consider this a conflict though...